tests/unit/test_configurator/conftest.py

```diff
@@ -196,6 +198,7 @@ def get_side_effect(param):
             "is_lambda": is_lambda,
             "extension": mock_ext,
             "oboe_api": mocker.Mock(),
+            "convert_to_bool": SolarWindsApmConfig.convert_to_bool,
```

Comment:
I am being lazy and keeping the elaborate test mocks but using the real `convert_to_bool` function to mock this function.